pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -395,6 +401,9 @@ func (w *watcher) execute(ctx context.Context, repo git.Repo, repoID string, eve
 					})
 					continue
 				}
+				if handler.Config.MakePullRequest {
+					newBranchs = append(newBranchs, branchName)
+				}
```

Comment:
To simplify this, let's make `newBranchs` a map instead of an array. In that case, whichever branch name is returned by `commitFiles` will be added to the `newBranchs` map. Then later, we can iterate through the map and push to those branches.